Makefile

```diff
@@ -257,8 +257,8 @@ $(SDK)/cmdline-tools/latest/bin/sdkmanager:
 sdk: $(SDK)/cmdline-tools/latest/bin/sdkmanager
 	yes y | $(SDK)/cmdline-tools/latest/bin/sdkmanager "platform-tools"
 	yes y | $(SDK)/cmdline-tools/latest/bin/sdkmanager "platforms;android-$(ANDROID_API)"
-	yes y | $(SDK)/cmdline-tools/latest/bin/sdkmanager "system-images;android-$(ANDROID_API);default;x86_64"
-	yes y | $(SDK)/cmdline-tools/latest/bin/sdkmanager "build-tools;30.0.3"
+	yes y | $(SDK)/cmdline-tools/latest/bin/sdkmanager "system-images;android-$(ANDROID_API);google_apis_playstore;x86_64"
```

Comment:
Switching from the barebones default image to the Google Play image is kind of independent, but it's probably the right thing to do as we do expect to be able to poke at the Play Store in a couple scenarios.

Reply:
I couldn’t find a `default` image for Android 33, so had to switch to something, and `google_apis_playstore` seemed like the best choice.

The available options for me are:
```sh
$ sdkmanager --list | grep system-images | grep android-33
  system-images;android-33;google_apis_playstore;x86_64 | 7            | Google Play Intel x86_64 Atom System Image | system-images/android-33/google_apis_playstore/x86_64
  system-images;android-33-ext4;google_apis_playstore;arm64-v8a                            | 1            | Google Play ARM 64 v8a System Image                                 
  system-images;android-33-ext4;google_apis_playstore;x86_64                               | 1            | Google Play Intel x86_64 Atom System Image                          
  system-images;android-33-ext5;google_apis_playstore;arm64-v8a                            | 1            | Google Play ARM 64 v8a System Image                                 
  system-images;android-33-ext5;google_apis_playstore;x86_64                               | 1            | Google Play Intel x86_64 Atom System Image                          
  system-images;android-33;android-tv;arm64-v8a                                            | 5            | Android TV ARM 64 v8a System Image                                  
  system-images;android-33;android-tv;x86                                                  | 5            | Android TV Intel x86 Atom System Image                              
  system-images;android-33;google-tv;arm64-v8a                                             | 5            | Google TV ARM 64 v8a System Image                                   
  system-images;android-33;google-tv;x86                                                   | 5            | Google TV Intel x86 Atom System Image                               
  system-images;android-33;google_apis;arm64-v8a                                           | 13           | Google APIs ARM 64 v8a System Image                                 
  system-images;android-33;google_apis;x86_64                                              | 13           | Google APIs Intel x86_64 Atom System Image                          
  system-images;android-33;google_apis_playstore;arm64-v8a                                 | 7            | Google Play ARM 64 v8a System Image                                 
  system-images;android-33;google_apis_playstore;x86_64                                    | 7            | Google Play Intel x86_64 Atom System Image
```

Reply:
Interesting, I have the same options. I'm pretty sure there's an `aosp` variant somewhere, but I think you must have to enable a separate SDK repository.